packages/@aws-cdk/aws-appsync/test/appsync-http.test.ts

```diff
@@ -57,6 +57,35 @@ describe('Http Data Source configuration', () => {
     });
   });
 
+  test('appsync configures name, authorizationConfig correctly', () => {
+    // WHEN
+    api.addHttpDataSource('ds', endpoint, {
+      name: 'custom',
+      description: 'custom description',
+      authorizationConfig: {
+        authorizationType: 'AWS_IAM',
+        awsIamConfig: {
+          signingRegion: 'us-east-1',
+          signingServiceName: 'states'
+        }
```

Comment:
I was misunderstanding your comment. It would be better to configure without the authorizationType because the authorizationType takes a single value.  Is it matched what you wanted to point it out?

Reply:
oh my bad.. i apologize i definitely misread this it LGTM